src/style.css

```diff
@@ -1 +1,64 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
I think that you know what a reset.css is, create `reset.css` file, and move this styles to it. Also it's a bad practice to reset styles with `*` selector, you should specify exact tags, that you are applying this styles to. You can also move a `list-style: none` for `ul` to this file